src/Feature/Changelog/Feature_Changelog.re

```diff
@@ -0,0 +1,402 @@
+          </View>;
+        };
+
+        let commit = (~item, ~uiFont, ~theme, ()) => {
+          <View style=Styles.breakingCommit>
+            <Text
+              text={item.summary |> Base.String.split(~on='\n') |> List.hd}
+              style={Styles.summary(uiFont, ~theme)}
+            />
+            {item.breaking
+             |> List.map(text => <breakingChange text uiFont theme />)
+             |> React.listToElement}
+          </View>;
+        };
+
+        let make = (~items, ~uiFont, ~theme, ()) => {
+          <View>
+            <Text
+              text="Breaking Changes"
+              style={Styles.groupHeader(uiFont, ~theme)}
+            />
+            <View style=Styles.groupBody>
+              {items
+               |> List.map(item => <commit item uiFont theme />)
+               |> React.listToElement}
+            </View>
+          </View>;
+        };
+      };
+
+      // FEATURES
+
+      module Features = {
+        let make = (~items, ~uiFont, ~theme, ()) => {
+          <View>
+            <Text
+              text="Features"
+              style={Styles.groupHeader(uiFont, ~theme)}
+            />
+            <View style=Styles.groupBody>
+              {items
+               |> List.map(commit => <line commit uiFont theme />)
+               |> React.listToElement}
+            </View>
+          </View>;
+        };
+      };
+
+      // FIXES
+
+      module Fixes = {
+        let make = (~items, ~uiFont, ~theme, ()) => {
+          <View>
+            <Text
+              text="Bugfixes"
+              style={Styles.groupHeader(uiFont, ~theme)}
+            />
+            <View style=Styles.groupBody>
+              {items
+               |> List.map(commit => <line commit uiFont theme />)
+               |> React.listToElement}
+            </View>
+          </View>;
+        };
+      };
+    };
+
+    // Update.make
+
+    let make = (~since, ~theme, ~uiFont, ()) => {
+      switch (Lazy.force(model)) {
+      | Ok(commits) =>
+        let commits =
+          List.fold_right(
+            commit =>
+              fun
+              | None when since == commit.hash => Some([])
+              | None => None
+              | Some(commits) => Some([commit, ...commits]),
+            commits,
+            None,
+          )
+          |> Option.value(~default=commits);
+        let breaking = commits |> List.filter(commit => commit.breaking != []);
+        let features =
+          commits |> List.filter(({typ, _}) => typ == Some("feat"));
+        let fixes =
+          commits
+          |> List.filter(
+               fun
+               | {typ: Some("fix" | "perf"), _} => true
+               | _ => false,
+             );
+
+        <ScrollView style=Styles.scrollContainer>
+          <View style=Styles.content>
+            <Parts.Breaking items=breaking uiFont theme />
+            <Parts.Features items=features uiFont theme />
+            <Parts.Fixes items=fixes uiFont theme />
```

Comment:
Trying out a new convention. The `Parts` module is meant for internal sub-components. I was hoping it would be possible to do `Parts.breaking`, but unfortunately Reason won't accept that.

Reply:
Looks nice! We could add this guidance to our coding standards. Seems reasonable to have a `Parts` module for internal sub components, and the `Styles` module for our internal styles.